scalar_udf.go

```diff
@@ -1,12 +1,11 @@
 package duckdb
 
 /*
-#include <duckdb.h>
+void scalar_udf_callback(void *, void *, void *);
+typedef void (*scalar_udf_callback_t)(void *, void *, void *);
 
-void scalar_udf_callback(duckdb_function_info, duckdb_data_chunk, duckdb_vector);
-void udf_delete_callback(void *);
-
-typedef void (*scalar_udf_callback_t)(duckdb_function_info, duckdb_data_chunk, duckdb_vector);
+void scalar_udf_delete_callback(void *);
+typedef void (*scalar_udf_delete_callback_t)(void *);
```

Comment:
Really ugly to just use `void *`, moving this to mapping would also be ugly I suppose. Just keep it this way?

Reply:
Yeah, I also don't like it... But I don't see a way around it. I prefer to keep the mappings what they are, i.e., just aliases, and not move any logic there.